fa.json

```diff
@@ -448,1247 +448,1247 @@
+				"option-encryption-custom-desc": "قوی‌ترین امنیت را ارائه می‌دهد اما نیاز دارد که رمز عبور رمزگذاری خود را به صورت ایمن ذخیره کنید",
+				"option-encryption-managed": "رمزگذاری استاندارد",
+				"option-encryption-managed-desc": "از یک کلید مدیریت شده توسط آبسیدین برای محافظت از داده‌های شما در حین انتقال و در سرورهای ما استفاده می‌کند.",
+				"label-sync-settings": "تنظیمات همگام‌سازی",
+				"label-sync-settings-desc": "آیا می‌خواهید هنگام همگام‌سازی با این دستگاه، پوشه‌ها یا فایل‌ها را مستثنی کنید؟",
+				"button-start": "شروع همگام‌سازی",
+				"label-sign-in-or-sign-up": "یک حساب آبسیدین ایجاد کنید یا وارد شوید",
+				"option-sign-up": "ثبت‌نام",
+				"option-sign-in": "ورود",
+				"button-resend-email": "ارسال مجدد ایمیل",
+				"label-remote-vault-options": "شما خزانه‌های همگام‌سازی موجود دارید",
+				"label-remote-vault-options-desc": "آیا می‌خواهید یک خزانه جدید ایجاد کنید یا به یک خزانه همگام‌سازی شده موجود متصل شوید؟",
+				"option-remote-create": "ایجاد خزانه همگام‌سازی جدید",
+				"option-remote-choose": "انتخاب یک خزانه موجود",
+				"button-logout": "خروج",
+				"button-forgot-password": "رمز عبور را فراموش کرده‌ام",
+				"label-vault-name": "نام خزانه",
+				"label-vault-location": "مکان خزانه",
+				"label-vault-location-desc": "خزانه جدید شما در {{vaultLocation}} قرار خواهد گرفت",
+				"label-region-selection": "منطقه",
+				"label-region-selection-help": "یک منطقه سرور نزدیک خود را انتخاب کنید تا همگام‌سازی سریع‌تر شود.",
+				"label-encryption-key": "کلید رمزگذاری",
+				"label-encryption-key-help": "این رمز عبور بعدا قابل تغییر نیست. اگر رمز عبور خود را فراموش کنید، هر گونه داده ریموت (از راه دور) در سرورهای Sync برای همیشه غیرقابل دسترسی خواهد بود.",
+				"placeholder-vault-name": "خزانه من",
+				"opt-region-selection-automatic": "خودکار",
+				"label-remote-vault-selection": "انتخاب یک خزانه",
+				"label-remote-vault-selection-desc": "حساب شما **{{email}}** خزانه‌های موجودی دارد که می‌توانید به آن‌ها متصل شوید.",
+				"label-sign-in": "ورود به حساب آبسیدین خود",
+				"label-forgot-password": "رمز عبور را فراموش کرده‌ام",
+				"button-reset-password": "بازنشانی رمز عبور",
+				"msg-password-reset": "ما ایمیلی به {{email}} برای بازنشانی رمز عبور شما ارسال کرده‌ایم.",
+				"label-sync-method-selection": "نحوه همگام‌سازی یادداشت‌های خود را انتخاب کنید",
+				"label-sync-method-selection-desc": "برای استفاده از آبسیدین در دستگاه‌های خود، باید داده‌های خود را همگام‌سازی کنید.",
+				"label-sync-pro1": "بدون نقص در هر دستگاه و سیستم‌عامل.",
+				"label-sync-pro2": "با قوی‌ترین استاندارد رمزگذاری سرتاسری ایمن شده است.",
+				"label-sync-pro3": "نیاز به یک حساب آبسیدین دارد.",
+				"label-icloud-con1": "فقط برای iOS و macOS.",
+				"label-icloud-con2": "به‌طور پیش‌فرض رمزگذاری نشده است.",
+				"label-third-party-desc1": "نیاز به افزونه‌ها یا ابزارهای شخص ثالث دارد.",
+				"label-third-party-desc2": "ممکن است به‌طور پیش‌فرض رمزگذاری نشده باشد.",
+				"opt-use-obsidian-sync": "استفاده از همگام‌سازی آبسیدین",
```

Comment:
Please don't translate the word "Obsidian"

Reply:
Please change them all to be "Obsidian"
"Obsidian" is a trademark, and we don't own any transliterations.

Reply:
Aha, thanks for the explanation. All done.